code/modules/mob/living/simple_animal/abnormality/waw/queen_bee.dm

```diff
@@ -48,6 +49,10 @@
 			if(prob(90))
 				var/datum/disease/bee_spawn/D = new()
 				H.ForceContractDisease(D, FALSE, TRUE)
+		for(var/mob/living/simple_animal/hostile/abnormality/general_b/Y in T.contents)
+			if(breached_others == FALSE)
+				Y.breach_effect()
```

Comment:
This will breach the general alone, without triggering zero qliphoth.

Reply:
Right, I know a solution.